README.md

```diff
@@ -0,0 +1,23 @@
+# matlab-travis-lang-test [![Build Status](https://staging.travis-ci.org/mathworks-continuous-integration/matlab-travis-lang-test.svg?branch=master)](https://staging.travis-ci.org/mathworks-continuous-integration/matlab-travis-lang-test)
+
+Repo for testing continuous integration support for MATLAB using Travis CI.
+
+## MATLAB? Using Travis CI?
+Indeed! It all starts with a minimal `.travis.yml` that looks like the following:
+
+```yaml
+language: matlab
+```
+
+The minimal configuration will run MATLAB tests in the repo (including subfolders) to determine if they all pass.
+
+MATLAB can also be invoked directly if custom commands are desired:
+```yaml
+language: matlab
+script: |
+    matlab -batch "disp("Hello, world!")"
+```
+_Invoke MATLAB using the '-batch' startup flag when running CI jobs_
+
+## See also:
+- [Continuous Integration (CI) - MATLAB & Simulink](https://www.mathworks.com/help/matlab/continuous-integration.html)
```

Comment:
Do you imagine this is temporary content that would go away when the docs are up? Should this repo just serve the one function of testing the language and leave teaching the language to the doc and example repos?

Reply:
Yeah I see this as only temporary content that does not replace the actual documentation. This repo really is for testing the language, but if someone happens upon it (since it is a public repo), I'm thinking this would set some context for them. 🤔 

Once the doc is up, I'll prolly strip away most of it and link straight to the doc 🤔

Reply:
You and @SangaviJayagopi made good points. I should just link Houman's PR for now and change it to the real doc when it's live 🤔